### DISCUSSION TITLE
Is it possible to configure kinematic objects, so they don't interact with each other?

### DISCUSSION THREAD

**Discussion by William-Godwin** [godot-jolt/godot-jolt#820] [Questions] [answered]
Is it possible to configure kinematic objects (in my case, player capsules) they don't interact with each other?
I don't need colliding capsules and also it reduces performance. So I want to find a way to only calculate collisions with static objects.
I looked through the documentation and did not find such functionality. Perhaps I was looking in the wrong place? =)

**Comment by mihe** [comment, marked as answer]
Have you tried using [collision layers and masks](https://docs.godotengine.org/en/stable/tutorials/physics/physics_introduction.html#collision-layers-and-masks)?

**Comment by William-Godwin** [comment]
Oh... I knew about layers, but I didn’t think that masking was also supported. Ok, sorry to bother you!
Thanks again for the answers! And for a cool extension! =)
